### PR TITLE
Add support to crash 7.1.6

### PR DESCRIPTION
**Pull request by fabianorosas**

### CODE REVIEW

**Comment by maurorodrigues** [comment]
Can one of the admins verify this patch?

File: crash/centOS/7.2/crash.spec [hunk 1]
```diff
@@ -0,0 +1,402 @@
+#
+# crash core analysis suite
+#
+Summary: Kernel analysis utility for live systems, netdump, diskdump, kdump, LKCD or mcore dumpfiles
+Name: crash
+Version: 7.1.6
+Release: 1%{?dist}
+License: GPLv3
+Group: Development/Debuggers
+Source: http://people.redhat.com/anderson/crash-%{version}.tar.gz
```
Comment (olavphibm) [inline crash/centOS/7.2/crash.spec]: Now that you've automated the version upgrades, it's probably better to remove the -%{version} from the tar file name. We're also checking out the git repository, which may differ from the tar.gz file available from this link.

**Comment by maurorodrigues** [comment]
please inform in the commit message the spec origin. 
Also, I'm guessing here, this is from fedora, why not from CentOS?

**Comment by fabianorosas** [comment]
The CentOS version does not work with the HostOS kernel. We need the latest crash version to support recent (4.4+) kernels.

**Comment by fabianorosas** [comment]
Test this please

**Comment by fabianorosas** [comment]
start tests

**Comment by maurorodrigues** [comment]
@fabianorosas this doesn't depend on open-power-host-os/versions#47, does it?

**Comment by fabianorosas** [comment]
@maurorodrigues no, it does not.

**Pull request merged**